pkg/curl/gen.lua

```diff
@@ -45,51 +48,48 @@ pkg.deps = {
-		hostip6.c hostsyn.c hsts.c http.c http2.c http_chunks.c
-		http_digest.c http_negotiate.c http_ntlm.c http_proxy.c
-		http_aws_sigv4.c idn_win32.c if2ip.c imap.c inet_ntop.c
-		inet_pton.c krb5.c ldap.c llist.c md4.c md5.c memdebug.c mime.c
-		mprintf.c mqtt.c multi.c netrc.c nonblock.c openldap.c
-		parsedate.c pingpong.c pop3.c progress.c psl.c rand.c rename.c
-		rtsp.c select.c sendf.c setopt.c sha256.c share.c slist.c smb.c
-		smtp.c socketpair.c socks.c socks_gssapi.c socks_sspi.c
-		speedcheck.c splay.c strcase.c strdup.c strerror.c strtok.c
-		strtoofft.c system_win32.c telnet.c tftp.c timediff.c timeval.c
-		transfer.c url.c urlapi.c version.c version_win32.c warnless.c
-		wildcard.c
+		altsvc.c amigaos.c asyn-ares.c asyn-thread.c base64.c bufq.c
+		bufref.c c-hyper.c cf-h1-proxy.c cf-h2-proxy.c cf-haproxy.c
+		cf-https-connect.c cf-socket.c cfilters.c conncache.c connect.c
+		content_encoding.c cookie.c curl_addrinfo.c curl_des.c
+		curl_endian.c curl_fnmatch.c curl_get_line.c curl_gethostname.c
+		curl_gssapi.c curl_memrchr.c curl_multibyte.c curl_ntlm_core.c
+		curl_ntlm_wb.c curl_path.c curl_range.c curl_rtmp.c curl_sasl.c
+		curl_sspi.c curl_threads.c curl_trc.c dict.c doh.c dynbuf.c
+		dynhds.c easy.c easygetopt.c easyoptions.c escape.c file.c
+		fileinfo.c fopen.c formdata.c ftp.c ftplistparser.c getenv.c
+		getinfo.c gopher.c hash.c headers.c hmac.c hostasyn.c hostip.c
+		hostip4.c hostip6.c hostsyn.c hsts.c http.c http1.c http2.c
+		http_aws_sigv4.c http_chunks.c http_digest.c http_negotiate.c
+		http_ntlm.c http_proxy.c idn.c if2ip.c imap.c inet_ntop.c
+		inet_pton.c krb5.c ldap.c llist.c macos.c md4.c md5.c memdebug.c
+		mime.c mprintf.c mqtt.c multi.c netrc.c nonblock.c noproxy.c
+		openldap.c parsedate.c pingpong.c pop3.c progress.c psl.c rand.c
+		rename.c rtsp.c select.c sendf.c setopt.c sha256.c share.c
+		slist.c smb.c smtp.c socketpair.c socks.c socks_gssapi.c
+		socks_sspi.c speedcheck.c splay.c strcase.c strdup.c strerror.c
+		strtok.c strtoofft.c system_win32.c telnet.c tftp.c timediff.c
+		timeval.c transfer.c url.c urlapi.c version.c version_win32.c
+		warnless.c ws.c
 		vauth/(
 			cleartext.c cram.c digest.c digest_sspi.c
 			gsasl.c krb5_gssapi.c krb5_sspi.c ntlm.c
 			ntlm_sspi.c oauth2.c spnego_gssapi.c
 			spnego_sspi.c vauth.c
 		)
 		vtls/(
-			bearssl.c gskit.c gtls.c hostcheck.c keylog.c mbedtls.c
-			mbedtls_threadlock.c nss.c openssl.c rustls.c schannel.c
+			bearssl.c gtls.c hostcheck.c keylog.c mbedtls.c
+			mbedtls_threadlock.c openssl.c rustls.c schannel.c
 			schannel_verify.c sectransp.c vtls.c wolfssl.c
 			x509asn1.c
 		)
-		vquic/(msh3.c ngtcp2.c quiche.c vquic.c)
+		vquic/(curl_msh3.c curl_ngtcp2.c curl_quiche.c vquic.c)
 		vssh/(libssh.c libssh2.c wolfssh.c)
 	)
 	$builddir/pkg/bearssl/libbearssl.a
 	$builddir/pkg/zlib/libz.a
 ]])
 
-build('cc', '$outdir/tool_hugehelp.c.o', {
-	'$dir/tool_hugehelp.c', '||', '$gendir/deps', '$srcdir/src/tool_hugehelp.h',
-})
-
```

Comment:
This is no longer necessary. If `MANUAL` is undefined there is a `#define hugehelp()` in `src/tool_hugehelp.h`.